code/datums/emergency_calls/upp_commando.dm

```diff
@@ -23,9 +23,10 @@
 	to_chat(M, SPAN_WARNING(FONT_SIZE_BIG("Glory to Colonel Ganbaatar.")))
 	to_chat(M, SPAN_WARNING(FONT_SIZE_BIG("Glory to the Smoldering Sons.")))
 	to_chat(M, SPAN_WARNING(FONT_SIZE_BIG("Glory to the UPP.")))
-	to_chat(M, SPAN_NOTICE(" Use say :3 <text> to speak in your native tongue."))
-	to_chat(M, SPAN_NOTICE(" This allows you to speak privately with your fellow UPP allies."))
+	to_chat(M, SPAN_NOTICE(" Use say :l <text> to speak in tactical sign language, understood only by similarly trained commando units."))
```

Comment:
though this will probably be correct, it won't always be. it'd be better to get the key like:

```dm
var/key = "1"
var/datum/language/commando/sign_language = locate() in M.languages
if(sign_language)
 key = sign_language.key
 ```
 
 then down here you'd just
```suggestion
	to_chat(M, SPAN_NOTICE(" Use say :[key] <text> to speak in tactical sign language, understood only by similarly trained commando units."))
```